### PR TITLE
[messages] Print warnings in info panel

### LINKED ISSUE #195: Display warnings in error browser
We should display warnings in the error browser, or at least make a new tab for them.

### PR DESCRIPTION
**Pull request by ejgallego**
Fixes #195

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -44,6 +44,8 @@
  - Goal display handles background goals better, showing preview,
    goals stack, and focusing information (@ejgallego, #290, fixes
    #288, based on jsCoq code by Shachar Itzhaky)
+ - Warnings are not printed in the info view messages panel
```
Comment (Alizter) [inline CHANGES.md:47]:
```suggestion
 - Warnings are now printed in the info view messages panel
```

File: controller/rq_goals.ml [hunk 2]
```diff
@@ -8,11 +8,12 @@
 (* Replace by ppx when we can print goals properly in the client *)
 let mk_message (_loc, lvl, msg) =
   let hdr =
-    if lvl = 4 then Pp.str "[info ] "
-    else if lvl = 5 then Pp.str "[debug] "
+    if lvl = 2 then Pp.str "[warning] "
+    else if lvl = 4 then Pp.str "[info   ] "
+    else if lvl = 5 then Pp.str "[debug  ] "
```
Comment (Alizter) [inline controller/rq_goals.ml:13]:
```suggestion
    else if lvl = 4 then Pp.str "[ info  ] "
    else if lvl = 5 then Pp.str "[ debug ] "
```

Comment (ejgallego) [reply controller/rq_goals.ml:13]: That does align poorly with `[warning]` , doesn' t it?

Comment (ejgallego) [reply controller/rq_goals.ml:13]: Let's send the level to the client instead.

**Pull request merged**